lib/periodicChecks.js

```diff
@@ -174,7 +175,57 @@ const ensureAllIssuesHaveProjects = async (context) => {
   });
 };
 
+/**
+ * This function comments on and tags pull requests in which the last
+ * commit was created 2 days ago
+ * @param {import('probot').Context} context
+ */
+const checkAndTagPRsWithOldBuilds = async (context) => {
+  const OLD_BUILD_LABEL = "PR: don't merge - STALE BUILD";
+
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+
+  allOpenPullRequests.forEach(async (pullRequest) => {
+    const containsOldBuildLabel = pullRequest.labels.some(
+      label => label.name === OLD_BUILD_LABEL
+    );
+    // If label has already been added, do nothing.
+    if (containsOldBuildLabel) {
+      return;
+    }
+    // Get last commit for pull request.
+    const {data: lastCommit} = await context.github.repos.getCommit(
+      context.repo({
+        ref: pullRequest.head.sha
+      })
+    );
+
+    const lastCommitDate = new Date(lastCommit.commit.author.date);
+    // Commit is older than 2 days
+    if (utilityModule.MIN_BUILD_DATE > lastCommitDate) {
+      const labelParams = context.repo({
+        issue_number: pullRequest.number,
+        labels: [OLD_BUILD_LABEL],
+      });
+      context.github.issues.addLabels(labelParams);
+
+      // Ping PR author.
+      const commentParams = context.repo({
+        issue_number: pullRequest.number,
+        body:
+            'Hi @' + pullRequest.user.login + ', the build of this PR is ' +
+            'stale and this could result in tests failing in develop. Please' +
+            ' update this pull request with the latest changes from develop,' +
+            ' Thanks.',
```

Comment:
Now that everything is shifted you should be able to do 
```suggestion
            'stale and this could result in tests failing in develop. Please' +
            'update this pull request with the latest changes from develop, ' +
            'Thanks. ',
```

Reply:
Done.